lib/teiserver/bridge/commands/findreports_command.ex

```diff
@@ -0,0 +1,113 @@
+defmodule Teiserver.Bridge.Commands.FindreportsCommand do
+  @moduledoc """
+  Calls the bot to link all connected reports discord messages
+  """
+  alias Teiserver.Bridge.{BridgeServer}
+  alias Teiserver.{Communication, Room, Logging, Config}
+  alias Teiserver.Moderation
+  import Ecto.Query
+  alias Teiserver.Repo
+  alias Teiserver.Moderation.Action
+
+  @behaviour Teiserver.Bridge.BridgeCommandBehaviour
+
+  @impl true
+  @spec name() :: String.t()
+  def name(), do: "findreports"
+
+  @impl true
+  @spec cmd_definition() :: map()
+  def cmd_definition() do
+    %{
+      name: "findreports",
+      description: "Find reports by the action message id",
+      options: [
+        %{
+          # type3 = String
+          type: 3,
+          name: "message_id",
+          description: "Message ID of the actions discord message",
+          required: true
+        }
+      ],
+      nsfw: false
+    }
+  end
+
+  @impl true
+  @spec execute(interaction :: Nostrum.Struct.Interaction.t(), options_map :: map()) :: map()
+  def execute(_interaction, options_map) do
+    message_id_str = options_map["message_id"]
+
+    content =
+      if message_id_str == nil do
+        "Please provide a message id"
+      else
+        case Integer.parse(message_id_str) do
+          {message_id, ""} ->
+            action =
+              from(a in Moderation.Action,
+                where: a.discord_message_id == ^message_id
+              )
+              |> Repo.one()
+
+            if action == nil do
+              "Please provide a valid message id"
+            else
```

Comment:
Not necessarily something that needs fixing, but it might be more informative to say that the bot was unable to find the action, which could happen if for example the action was deleted even if the message ID is valid.